sdk/resourcemanager/Proto.Client/compute/VirtualMachineScaleSetBuilder.cs

```diff
@@ -11,8 +11,10 @@ namespace Proto.Compute
     /// <summary>
     /// A class representing a builder object used to create Azure resources.
     /// </summary>
-    public class VirtualMachineScaleSetBuilder
+    public partial class VirtualMachineScaleSetBuilder
```

Comment:
why do we need this partial it appears both classes are in the same dll

Reply:
One of them is intended to be hand written and exist outside of generated folder.

Reply:
ah, yeah forgot about that